types/nuxtjs__auth/index.d.ts

```diff
@@ -47,7 +47,7 @@ export interface Auth<T = any> {
   strategy(): string;
   registerStrategy(strategyName: string, strategy: object): void;
   setStrategy(strategyName: string): void;
-  setUserToken(): Promise<never>;
+  setUserToken(token: string): Promise<never>;
```

Comment:
`Promise<never>` does not seem like the right type (unless the Promise is guaranteed to throw). I would expect this to be `Promise<void>` if its a promise whose value is unused.

Reply:
@rbuckton Thanks for your reviewing! I hadn't noticed that. Now, I've just fixed this type definition.